### ISSUE TITLE
`staticmethod`s are not treated as class members, instead they are treated as module members

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#350]
# Bug report

## What's wrong

```python
class Test(object):
     @staticmethod
     def method(): ...
```

This code will raise a warning with `--max-module-members` set to `1`.
But, it should not.

## How is that should be

It should not raise a module level warning.
`@staticmethod` should be treated as a class member.
